packages/winsvc/winsvc.1.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "Library to make OCaml program act as a Windows service"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
+homepage: "https://github.com/savonet/ocaml-winsvc"
+bug-reports: "https://github.com/savonet/ocaml-winsvc/issues"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {>= "4.12"}
```
Looking at your local CI it seems that only OCaml 4.12 is being tested. Do you know if it works with previous versions? @MisterDA Could you have a look on what's the expected lower bound constraint for this package maybe?

Reply:
`>= ocaml 4.06` seems fine, as we use OCaml Windows Unicode C functions that were introduced at that time, and everything should be older.

Reply:
```suggestion
depends: [
  "ocaml" {>= "4.06"}
```
Can you make a brief test with 4.06 in your local CI before we add the bound?

Reply:
I have tested and can confirm that winsvc builds fine with 4.06, and won't build with earlier versions.

Reply:
Thanks a lot!